src/codeGenerators/circuit/zokrates/toCircuit.ts

```diff
@@ -116,6 +116,25 @@ function codeGenerator(node: any) {
     case 'TupleExpression':
       return `(${node.components.map(codeGenerator).join(` `)})`;
 
+    case 'IfStatement': 
+      let trueStatements: any = ``;
+      let falseStatements: any= ``;
+      let initialStatements: any= ``;
+      console.log('expressios' , node.condition);
+      initialStatements+= `
+        // if statements start , copies over left expression variable to temporary variable
+        field ${node.condition.leftExpression.name}_temp = ${node.condition.leftExpression.name}`;
+      node.condition.leftExpression.name+= '_temp';
+      for (let i =0; i<node.trueBody.length; i++) {
+        trueStatements+= `
+        ${node.trueBody[i].expression.leftHandSide.name} = if ${codeGenerator(node.condition)} then ${codeGenerator(node.trueBody[i].expression.rightHandSide)} else ${node.trueBody[i].expression.leftHandSide.name}`
```

Comment:
At the end of true and false statements, I think we need to add `fi`, e.g. to make `z = if x_temp > 5 then z + y else z fi` (I added this manually in the example zapp and it worked)